Shoko.Server/API/v3/Models/Shoko/WebUI.cs

```diff
@@ -201,10 +201,10 @@ public WebUISeriesFileSummary(SVR_AnimeSeries series, HashSet<EpisodeType> episo
                 .DistinctBy(anidbFile => anidbFile.Hash)
                 .ToDictionary(anidbFile => anidbFile.Hash);
             var releaseGroups = anidbFiles.Values
-                .Select(anidbFile => anidbFile.AniDB_FileID)
+                .Select(anidbFile => anidbFile.ReleaseGroup.GroupID)
```

Comment:
If you're going to use the object helper instead of the id on the anidb file model, then you might as well rewrite it to select the model here, then distinct by the group id, then filter on the model for a group id of 0, then drop the select step after that.

Reply:
the alternative is to leave most of it as is, and just select the group id on the anidb file model so we don't do the group call early (and save a lot of repo lookups like we do currently).

Reply:
actually, lets go with that, since it will save on db calls (or more accurately, won't introduce _more_ db calls, even if we're just hitting the cache).